src/PowerShellEditorServices.Protocol/Server/DebugAdapter.cs

```diff
@@ -427,20 +426,27 @@ protected void Stop()
 
                     return;
                 }
+            }
+            else if (customPipeNameIsSet)
```

Comment:
Do all of the clauses above return? If so, is it possible to remove this `else`?

Reply:
They don't all return. We have to keep this else in.